test/data/ansible/check_ostree.yaml

```diff
@@ -515,7 +515,7 @@
           set_fact:
             failed_counter: "{{ failed_counter | int + 1 }}"
       # NOTE: disable on RHEL 9.0 -- failing for unknown reason -- see rhbz#2053469
-      when: result_rollback is succeeded and ((ansible_facts['distribution'] != 'RedHat' or ansible_facts['distribution_version'] | int != 9.0))
+      when: result_rollback is succeeded and (ansible_facts['distribution'] != 'RedHat' or ansible_facts['distribution_version'] is version('9', '!='))
```

Comment:
Huh. I was just about to comment that `int` should be `9`, not `9.0`. Good catch.